server/docs/source/index.html.md

```diff
@@ -428,7 +429,7 @@ Update a node details.
 
 ### Endpoint
 
-`PUT /v1/nodes/{node_id}`
+`PUT /v1/nodes/{id}`
```

Comment:
Inconsistent use of `:id` vs. `{id}`, although all the endpoints in the API docs seem to have these used inconsistently :(